server/chat-commands/core.ts

```diff
@@ -1177,7 +1177,8 @@ export const commands: ChatCommands = {
 		const name = this.targetUsername;
 
 		if (!targetUser) return this.errorReply(this.tr`User ${name} not found.`);
-		if (!targetUser.inRooms.has(room.roomid)) {
+		const targetRooms = targetUser.getRooms();
+		if (!targetRooms.has(room.roomid)) {
```

Comment:
```suggestion
		if (!(targetUser.id in room.users)) {
```
I'm concerned you didn't do this. You could write a `targetUser.inRoom(room)` function that does this, if you wanted it to be shorter, I think.